docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
       - SITE_URL=http://backend:8000/
       - TREEHERDER_DEBUG=True
       - NEW_RELIC_INSIGHTS_API_KEY=${NEW_RELIC_INSIGHTS_API_KEY:-}
+      - PROJECTS_TO_INGEST=${PROJECTS_TO_INGEST:-autoland,try}
```

Comment:
This seems to be key, otherwise you won't get all tasks even if you run the celery commands outlined in 'Running the ingestion tasks'.